frontend/aml_dashboard/src/index.js

```diff
@@ -0,0 +1,746 @@
+post_data.title = 'Context Broker Data';
+
+const upload_data = imageUpload();
+upload_data.title = 'Upload data to the Context Broker';
+
+const data_status = text('Not Sended');
+data_status.title = 'Data Status';
+
+const get_solution = text('{}');
+get_solution.title = 'Context Broker Solution';
+
+const solution_status = text('Not Received');
+solution_status.title = 'Solution Status';
+
+let image;
+
+const url_context_broker = "http://localhost:5000/context_broker/";
+
+// Create the Fiware Node
+create_fiware.$click.subscribe( async () => {
+
+  fiware_node_status.$value.set(' <h1>Creating...</h1> <br> <img src="https://i.gifer.com/Cad.gif">');
+
+  const parameters = {};
+  for (let param in fiware_params.parameters) {
+    parameters[param] = fiware_params.parameters[param].value;
+  }
+
+  fetch(url_context_broker + 'fiware', {
+    method: "POST", // *GET, POST, PUT, DELETE, etc.
+    mode: "cors", // no-cors, *cors, same-origin
+    cache: "no-cache", // *default, no-cache, reload, force-cache, only-if-cached
+    credentials: "omit", // include, *same-origin, omit
+    headers: {
+      "Content-Type": "application/json",
+    },
+    redirect: "follow", // manual, *follow, error
+    referrerPolicy: "no-referrer", // no-referrer, *no-referrer-when-downgrade, origin, origin-when-cross-origin, same-origin, strict-origin, strict-origin-when-cross-origin, unsafe-url
+    body: JSON.stringify(parameters), // body data type must match "Content-Type" header
+  })
+  .then(response => response.json())
+  .then(async json => {
+    console.log(json);
+    console.log('CREATED');
+
+    fiware_node_status.$value.set('<h1>Created !</h1> ');
+
+  });
+});
+
+// Post the data to the Context Broker
+post_data.$click.subscribe( async () => {
+
+  if (upload_data.$images.value == undefined)
+  {
+    throwError(new Error('No data has been uploaded'));
+  }
+  else
+  {
+    data_status.$value.set(' <h1>Sending...</h1> <br> <img src="https://i.gifer.com/Cad.gif">');
+    const image_data = Array.from(image.data);
+
+    fetch(url_context_broker + 'data', {
+      method: "POST", // *GET, POST, PUT, DELETE, etc.
+      mode: "cors", // no-cors, *cors, same-origin
+      cache: "no-cache", // *default, no-cache, reload, force-cache, only-if-cached
+      credentials: "omit", // include, *same-origin, omit
+      headers: {
+        "Content-Type": "application/json",
+      },
+      redirect: "follow", // manual, *follow, error
+      referrerPolicy: "no-referrer", // no-referrer, *no-referrer-when-downgrade, origin, origin-when-cross-origin, same-origin, strict-origin, strict-origin-when-cross-origin, unsafe-url
+      body: JSON.stringify({
+        width : 6,
+        height : 6,
+        data : image_data
+      }) // body data type must match "Content-Type" header
+    })
+    .then(response => response.json())
+    .then(async json => {
+      console.log('SENDED');
+
+      data_status.$value.set('<h1>Sended !</h1> ');
```

Comment:
Please add the case for when data could not be sent.


```suggestion
      data_status.$value.set('<h1>Data could not be sent !</h1> ');
```

Reply:
Addressed in #2